tests/mock_tests/mock_orchagent_main.h

```diff
@@ -29,6 +29,9 @@
 #include "nhgorch.h"
 #include "copporch.h"
 #include "twamporch.h"
+#define private public
```

Comment:
why was this required ?

Reply:
This is added to access spOrch class APIs here in the file